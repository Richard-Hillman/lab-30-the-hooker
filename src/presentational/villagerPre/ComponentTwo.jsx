// villager
/* eslint-disable max-len */
import React from 'react';
import PropTypes from 'prop-types';
// import styles from './ComponentOne.css';


const Villager = ({ name, japaneseName, image, quote }) => {
  return ( 
    <figure data-testId>
      <img src={image} alt={name} />
      <figcaption>
        <p>{name}</p>
        <p>{japaneseName}</p>
        <p>{quote}</p>
      </figcaption>
    </figure>
  );
};

Villager.propTypes = {
  name: PropTypes.string.isRequired,
  japaneseName: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
  quote: PropTypes.string.isRequired,
};

export default Villager;
