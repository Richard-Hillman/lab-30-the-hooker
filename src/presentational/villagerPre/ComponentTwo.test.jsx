import React from 'react';
import { render, cleanup } from '@testing-library/react';
import ComponentTwo from './ComponentTwo';

describe('ComponentTwo component', () => {
  afterEach(() => cleanup());
  it('renders ComponentTwo', () => {
    const { asFragment } = render(<ComponentTwo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
