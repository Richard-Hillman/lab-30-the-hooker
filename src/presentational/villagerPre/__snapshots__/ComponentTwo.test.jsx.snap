// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentTwo component renders ComponentTwo 1`] = `
<DocumentFragment>
  <figure
    data-testid="true"
  >
    <img />
    <figcaption>
      <p />
      <p />
      <p />
    </figcaption>
  </figure>
</DocumentFragment>
`;
